### DISCUSSION TITLE
Progress Report 9 – 13 May

### DISCUSSION THREAD

**Discussion by esheehan-gsl** [NOAA-GSL/unified-graphics#18] [Progress Report]
🚴🏻 **Progress**
- #16 is being reviewed

😖 **Regressions**
- None

💬 **To Discuss**
- Dev environment with Kubernetes (see #16 for some of this conversation)

📆 **Next Week**
- Scaffold the backend Python infrastructure
- Read data out of NetCDF diag files

**Comment by ian-noaa** [comment]
🚴🏻 **Progress**
- VLab is working on creating our EKS cluster
  - [To meet their security and monitoring requirements, they have created a custom AMI for the cluster worker nodes with Packer](https://vlab.noaa.gov/redmine/issues/104656).
  - [They are working on creating an EKS cluster using those worker nodes](https://vlab.noaa.gov/redmine/issues/104724).
  - [They generated a monthly cost estimate for our resources](https://vlab.noaa.gov/redmine/issues/104426#note-4).

😖 **Regressions**
- None

💬 **To Discuss**
- Ran into issues running Rancher Desktop on Ubuntu and am considering alternatives

📆 **Next Week**
- Help with scaffolding the backend Python infrastructure
- Look into setting up GitHub CI for the frontend
- Continue exploring Argo CD for deployments
